### PR TITLE
Simplify

### PR DESCRIPTION
**Pull request by bbannon**
Redundant variables.

### CODE REVIEW

File: includes/functions.php [hunk 1]
```diff
@@ -5504,16 +5506,20 @@ function send_doctype ( $doc_title = '' ) {
...
 
-  return '<?xml version="1.0" encoding="' . $charset . '"?' . '>
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
-  "DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="' . $lang . '" lang="'
-   . $lang . '">
+// Craig. This is the major part of converting to HTML5.
+// The rest:
+//    changing " />" to ">" - on "<br />" "<input... />", etc.
+//    removing "<!-- <![CDATA[" and "//]]> -->"
```
Comment (craigk5n) [inline includes/functions.php]: This is fine by me....

File: includes/functions.php [hunk 2]
```diff
@@ -6186,7 +6192,28 @@ function build_entry_popup ( $popupid, $user, $description = '', $time,
...
+ *
+ * param  string  name of file
+ * param  bool    strip comments or not
+ */
+function file2array ( $file, $nocomments = 0 ) {
+  $ret = [];
+  foreach ( file ( $file, FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES ) as $l ) {
+    // Strip php open/close and comments?
+    if ( $nocomments && preg_match ( '/^\s*(<\?|#|\/\*|\*|\?>)/', $l ) )
+      continue;
```
Comment (craigk5n) [inline includes/functions.php]: The idea seems good... might want to unit test the function to verify it works as expected.

Comment (bbannon) [reply includes/functions.php]: It worked in my tests. No problems.

File: includes/functions.php [hunk 3]
```diff
@@ -5504,16 +5503,13 @@ function send_doctype ( $doc_title = '' ) {
...
 
-  $charset = ( empty ( $LANGUAGE ) ? 'iso-8859-1' : translate ( 'charset' ) );
+  $charset = ( empty ( $LANGUAGE ) ? 'utf-8' : translate ( 'charset' ) );
 
-  return '<?xml version="1.0" encoding="' . $charset . '"?' . '>
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
-  "DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="' . $lang . '" lang="'
-   . $lang . '">
+  return '<!DOCTYPE html>
```
Comment (craigk5n) [inline includes/functions.php:5508]: This change has already been made (conflict).

**Pull request closed without merging**